tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -39,6 +39,10 @@ public class ImageTests
                 new ImageDescriptor { RuntimeVersion = "4.7.2", BuildVersion = "4.7.2", OsVariant = WSC_LTSC2016 },
                 new ImageDescriptor { RuntimeVersion = "4.7.2", BuildVersion = "4.7.2", OsVariant = WSC_1803 },
                 new ImageDescriptor { RuntimeVersion = "4.7.2", BuildVersion = "4.7.2", OsVariant = WSC_LTSC2019 },
+                new ImageDescriptor { RuntimeVersion = "4.6.2", BuildVersion = "4.8", OsVariant = WSC_LTSC2016 },
```

Comment:
Please group these new entries by the RuntimeVersions so that it is easier to read.